docs/howto/static-files/index.txt

```diff
@@ -137,6 +137,22 @@ this by adding the following snippet to your urls.py::
     the given prefix is local (e.g. ``/media/``) and not a URL (e.g.
     ``http://media.example.com/``).
 
+Customizing the ignored pattern list
+====================================
+
+When collecting files through the :djadmin:`collectstatic` management command,
+files matching the pattern list ``['CVS', '.*', '*~']`` are ignored by default.
+If you need to customize this list on a more stable way than providing the
+``--ignore`` command option at each invocation, you can provide a custom
+:class:`~django.apps.AppConfig` class, override the ``ignore_patterns``
+attribute of this class and specify that class path inside your
+:setting:`INSTALLED_APPS` setting::
+
+    from django.contrib.staticfiles.apps import StaticFilesConfig
+
+    class IgnorePatternsAppConfig(StaticFilesConfig):
+        ignore_patterns = [<your custom ignore list>]
```

Comment:
Syntax highlighting is failing due to invalid syntax. maybe something like, `= [...]  # your custom ignore list` would work.